modules/Bio/EnsEMBL/Variation/Utils/VEP.pm

```diff
@@ -933,12 +924,7 @@ sub convert_to_vcf {
     else {
 
         # convert to SO term
-        my %terms = (
-            'insertion' => 'INS',
-            'deletion' => 'DEL',
-            'tandem_duplication' => 'TDUP',
-            'duplication' => 'DUP'
-        );
+        my %terms = reverse %SV_SO_TERMS;
```

Comment:
The ALT format in VCF format does not allow some of the items from `%terms` (according to [VCF 4.4 specs](https://samtools.github.io/hts-specs/VCFv4.4.pdf)). VCF only allows SV symbolic alleles in ALT starting with the following terms:
* DEL
* INS
* DUP
* INV
* CNV

Instead of modifying `%terms` (may have potential issues?), we could add logic here to simply convert:
- `TDUP` to `DUP:TANDEM`
- `TREP` to `CNV:TR`
- `BND` to the alternative allele itself
- `INS_ALU` to `INS:ME:ALU` (and the same for the other cases for mobile element insertion/deletion); maybe it would be easier to add a `ME_` to the key of all mobile elements and replace `_` with `:`?